packages/assets-controllers/package.json

```diff
@@ -41,8 +41,8 @@
     "@metamask/controller-utils": "^5.0.2",
     "@metamask/eth-query": "^4.0.0",
     "@metamask/metamask-eth-abis": "3.0.0",
-    "@metamask/network-controller": "^15.2.0",
-    "@metamask/polling-controller": "^1.0.1",
+    "@metamask/network-controller": "^16.0.0",
+    "@metamask/polling-controller": "^1.0.2",
```

Comment:
Should we add this to the changelog too?

Reply:
Good catch! I think we should include it

Reply:
Fixed here: https://github.com/MetaMask/core/pull/2044/commits/194c74e747285515e41738387b9b59d156b9b0fe! I'll have to watch out for those concurrent version bumps. It would be nice to have `create-release-branch` add these to the changelogs automatically.

Reply:
Agreed! We have a ticket for this filed here: https://github.com/MetaMask/create-release-branch/issues/108